src/modules/layout/containers/__tests__/__snapshots__/LocationLayout.spec.js.snap

```diff
@@ -23,15 +23,15 @@ exports[`LocationLayout should show CategoriesToolbar if current route is catego
         Array [
           CityModel {
             "_code": "city1",
-            "_eventsEnabled": undefined,
-            "_extrasEnabled": undefined,
-            "_live": undefined,
+            "_eventsEnabled": true,
+            "_extrasEnabled": false,
+            "_live": true,
             "_name": "Mambo No. 5",
-            "_sortingName": undefined,
+            "_sortingName": "Mambo",
           },
         ]
       }
-      feedbackType={undefined}
+      feedbackStatus={undefined}
```

Comment:
feedbackStatus should never be undefined. Sth's wrong with the test.

Reply:
Yes it should, as this is a query param which isn't always there. We display the modal only if it is there.

Reply:
But in the PropsType of FeedbackModal, you defined, that `feedbackStatus` is always given a string-value (which can't be undefined)

Reply:
So the connect() thingy is breaking flow to recognize the PropsType properly

Reply:
Yep you're right. I think LocationLayout shouldn't be connected to the state anyway. But I think the problem here is that we don't have flow types for query params in state...